front/src/utils/timeManipulation.ts

```diff
@@ -87,6 +87,10 @@ export function ISO8601Duration2sec(isoDuration: string) {
   return durationDictionnary.asSeconds();
 }
 
-export function getStopTime(sec: number) {
-  return new Date(sec * 1000).toISOString().substr(11, 5);
+export function secToHoursString(sec: number | null, withSeconds = false) {
```

Comment:
Could use a comment, its purpose is not entirely clear. And I believe it's missing a type annotation for the return? (though that seems common in the code base, maybe it's normal)

Reply:
done.
we don’t usually specify the return type annotation as it’s inferred. Here we have an alias, so we might as well use it for the return type.

Reply:
Thanks, it's easier to understand :+1: 

But now that I get the intent, I feel like something like `d3.timeFormat('%H:%M')` would make it easier to follow (it's used in other places in the file).

But feel free to ignore this if you're running out of time, I realize I'm not making things go faster here

Reply:
done